src/docs/Components/Accordion/previews/selectable.preview.html

```diff
@@ -0,0 +1,50 @@
+<div class="example">
+  <div class="mc-accordion mc-accordion--s">
+    <input type="checkbox" id="icon" class="mc-accordion__trigger" />
+    <label
+      for="icon"
+      class="mc-accordion__label mc-accordion__label--selectable"
+      aria-expanded="false"
+    >
+      <div class="mc-checkbox mc-accordion__selectable">
+        <input
+          id="checkbox-01"
+          type="checkbox"
+          class="mc-checkbox__input"
+          name="example"
+        />
+      </div>
+      Accordion with checkbox
+    </label>
```

Comment:
Unfortunately, this HTML structure is not semantic.

I think the easiest way would be to use the structure you made on Mozaic-Vue => https://github.com/adeo/mozaic-vue/blob/master/packages/vue/src/components/accordion/MAccordion.vue